### DISCUSSION TITLE
What are the actual state values for listings, I've seen "edit" which isn't documented

### DISCUSSION THREAD

**Discussion by seandmccarthy** [etsy/open-api#1051] [Q&A] [answered]
I've used the API to create a draft listing, then updated it's state to "active", then updated its state again to "inactive". When I use getListing to retrieve the state of the listing, it shows as "edit" instead of the expected "inactive" value. I notice the same occurs if I use the Etsy UI to "deactivate" a listing.

I don't see "edit" mentioned as a state value anywhere in the listing APIs. Can I expect it to be this value whenever I set a listing to "inactive", or should check for both "edit" and "inactive" when trying to deal with listings that are deactivated by API or UI? Can the documentation be updated to accurately reflect what the states are?

**Comment by emergingdzns** [comment, marked as answer]
The "edit" state is basically synonymous with "inactive". It just means that the listing was live and then set to inactive. The api will sometimes show "edit" as the state. But if you are pulling listings and want to get inactive listings, use "inactive". It's a bit confusing.
